eks-anywhere-snow/Addons/Core/storage-driver/storage-driver.yaml

```diff
@@ -1,10 +1,24 @@
+# apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
```

Comment:
why do you need the commented out section? let's drop it.

Reply:
Removed the commented sec.